src/extensions/spikeamplitudes/SpikeAmplitudesView/SpikeAmplitudesPanel.ts

```diff
@@ -0,0 +1,155 @@
+import { Brush, CanvasPainter, Font, Pen } from "../../CanvasWidget/CanvasPainter"
+import { HitherContext, Recording, Sorting } from "../../extensionInterface"
+
+// const calculationPool = createCalculationPool({maxSimultaneous: 6})
+
+const colorList = [
+    'blue',
+    'green',
+    'red',
+    'orange',
+    'purple'
+]
```

Comment:
Might need a longer list, given the number of units. Also we should look into choosing accessible palettes--we'd like the tool to be useful including to color-blind or low-vision users. I just spent about half an hour looking for something off-the-shelf for this and haven't found it yet, so we should probably just open up a ticket for it and do it later...

(ref https://blog.graphiq.com/finding-the-right-color-palettes-for-data-visualizations-fcd4e707a283 https://medium.com/carbondesign/color-palettes-and-accessibility-features-for-data-visualization-7869f4874fca and many others)

Reply:
yeah this is important. We need a good, consistent way to generate colors for units, and colors for channels.